util_scripts/Populate_Metadata.py

```diff
@@ -85,15 +61,16 @@ def populate_metadata(client, conn, script_params):
             description="Choose source of images",
             values=dataTypes, default="Plate"),
 
-        scripts.String(
+        scripts.Long(
```

Comment:
The current implementation works only for a single object: https://github.com/emilroz/omero-user-scripts/blob/populate_metadata_fa_additions/util_scripts/Populate_Metadata.py#L35

That's mainly because the metadata is usually associated with a single screen or plate. I can change it to work for multiple objects if needed.